### DISCUSSION TITLE
is only upload possible?

### DISCUSSION THREAD

**Discussion by Jumbo125** [abraunegg/onedrive#2704] [Q&A] [answered]
Hello

Looks like a really great project. Thanks for your work!!!

I looking for a solution to uploads file to onedrive. My idea was to do that with php. But the authentication is really confused. 
So I logged into azure and created a app to get client id and more. It is so much uncomfortable....

Now I found this work. 
I want to upload files. After uploaded, the file should be delete on the local system.
Is it possible, only to upload file and not save ot locally

Thank you

**Comment by abraunegg** [comment, marked as answer]
@Jumbo125 
Please read the documentation - this should be your first port of call.

**Reply by Jumbo125** [reply]
Did it during last night. 
I think, that i found it
onedrive --synchronize --upload-only --no-remote-delete

Now it would be interested how I can detetct if it is uploaded...
It would be great, when I copy a file into the onedrive directory and after uploaded it would be removed locally

**Reply by abraunegg** [reply]
Again, read the documentation:
- run a systemd service that uses --monitor to watch the local directory, adding upload only + no remote delete directives
- enable logging to a file
- enable verbose logging

Read the logs into Splunk or similar, generate an alert or OOB  notification to say X occured
